packages/main/src/Popup.js

```diff
@@ -333,7 +351,7 @@ class Popup extends UI5Element {
 			this._removeOpenedPopup();
 		}
 
-		if (!preventFocusRestore) {
+		if (!this.preventFocusRestore && !preventFocusRestore) {
```

Comment:
Not sure if we need the third parameter to close any more. What do you think?

Reply:
Currently the TextArea, the Input and InputSuggestions classes make calls like: close(false, false, true). 

I think there are cases we want the focus to be restored, and to prevent it on some interactions within the TextArea and the Input. On the other hand to make this quick card view popover behave as expected it should have this prevent-focus-restore always.

But, I will definitely check if it is needed.

Reply:
I managed to replace two calls of "close(false, false, true)" in the TextArea and the Input, both related to the valueStateMsg popover, But one in the InputSuggestion is conditional:

Here we should restore the focus, when suggestion is selected, otherwise - prevent it.
```js
this.Suggestions.toggle(shouldOpenSuggestions, {
   preventFocusRestore: !this.hasSuggestionItemSelected,
});
```